istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -50,7 +50,7 @@ data:
     {{- else }}
     mixerCheckServer: istio-policy.{{ .Values.global.policyNamespace }}.svc.cluster.local:9091
     {{- end }}
-    {{- if and .Values.pilot.telemetry.enabled .Values.pilot.telemetry.v1.enabled }}
+    {{- if and .Values.telemetry.enabled}}
```

Comment:
remove `and`?

Reply:
@jwendell I am not sure what the desired approach here was, but master should be fixed first if your suggestion is correct.

Reply:
@sdake it's fixed in master. I think that was a typo/leftover when backporting to 1.4.